src/frontend/discussions-abusefilter-app/src/interfaces/BackendInterface.ts

```diff
@@ -0,0 +1,9 @@
+import type { Filter, FilterMetadata } from '@shared/filters';
+
+export interface BackendInterface {
+  getFilters: <T = any>() => Promise<T> | FilterMetadata[],
```

Comment:
```suggestion
  getFilters: () => Promise<FilterMetadata[]>,
```
The `LocalBackendInterface` functions should all return a promise that just resolves with the data instantaneously, so that the interface is the same. That includes the create/update/deleteFilter